src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/VsService`1.cs

```diff
@@ -32,7 +32,7 @@ public VsService([Import(typeof(SAsyncServiceProvider))]IAsyncServiceProvider se
             _threadingService = threadingService;
         }
 
-        public Task<T> GetValueAsync()
+        public virtual Task<T> GetValueAsync()
```

Comment:
Presumably this, and perhaps the special Mock helper, can go away if everything used IVsService